datawell/iex.py

```diff
@@ -56,3 +57,28 @@ def load_from_iex(self, uri: str):
             error = response.status_code
             self.Logger.error(
                 f'Encountered an error: {error} ( {response.text} ) while retrieving {uri}')
+    
+    def getCompany(self, *Symbols):
```

Comment:
Make sure to explain * and ** to your colleagues, ok?

Reply:
sure!